google/auth/compute_engine/credentials.py

```diff
@@ -71,7 +70,7 @@ def _retrieve_info(self, request):
             service_account=self._service_account_email)
 
         self._service_account_email = info['email']
-        self._scopes = _helpers.string_to_scopes(info['scopes'])
+        self._scopes = info['scopes']
```

Comment:
Yeah, the info endpoint returns the scopes as a list, not a string like tokeninfo/oauth2.